src/main.rs

```diff
@@ -151,6 +152,12 @@ enum Subcommand {
     /// This token can also be configured by editing ~/.foreman/auth.toml.
     #[structopt(name = "gitlab-auth")]
     GitLabAuth(GitLabAuthCommand),
+
+    /// Create a path to publish to artifactory
+    ///
+    /// Foreman does not support uploading the artifactory itself but will generate the path it expects to find artifacts.
```

Comment:
```suggestion
    /// Foreman does not support uploading binaries to artifactory directly, but it can generate the path where it would expect to find a given artifact. Use this command to generate paths that can be input to generic artifactory upload solutions.
```